itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/TestAcidOnTez.java

```diff
@@ -18,6 +18,7 @@
 
 package org.apache.hadoop.hive.ql;
 
+import static org.apache.hadoop.hive.ql.TxnCommandsBaseForTests.*;
```

Comment:
nit: do not use wildcard imports

Reply:
Is there any other change in this file than the import change? If not I would revert this file

Reply:
import is required after the tests refactor, previous visibility was package-private.
ps: moved tests refactor under https://github.com/apache/hive/pull/2831